package.json

```diff
@@ -271,6 +271,7 @@
     "@elastic/filesaver": "1.1.2",
     "@elastic/makelogs": "^6.1.0",
     "@faker-js/faker": "^7.6.0",
+    "@lhci/cli": "^0.14.0",
```

Comment:
if this is a dev dependency then bootstrapping should be suffice enough correct? do we still need to add it the step to install it globally in the github workflow?